code/datums/diseases/_MobProcs.dm

```diff
@@ -41,6 +41,11 @@
 		viruses += DD
 		DD.affected_mob = src
 		SSdisease.active_diseases += DD //Add it to the active diseases list, now that it's actually in a mob and being processed.
+		if(istype(DD, /datum/disease/advance)) //Log the infection
+			var/datum/disease/advance/A = DD
+			SSblackbox.add_details("infections", "[A.name] - Symptoms: [english_list(A.symptoms)]")
```

Comment:
Is there any way you can standardize the order symptoms are displayed in so that when you have two viruses, one with Coughing and Sneezing and the other with Sneezing and Coughing they generate the same feedback string?

Reply:
Virus IDs do that, but they aren't readable. I could swap the numbers for two-letter codes or something.

Reply:
While you're here I'd remove the `Symptoms: `, unnecessary extra text while `-` already works as a delimiter.

Reply:
Looking at GetDiseaseID(), it may be possible to sort the symptoms if you do something like this after every AddSymptom()

```
var/list/L = list()
var/list/_symptoms = symptoms
for(var/I in symptoms)
	var/datum/symptom/S = I
	L[S.id] = S
_symptoms.Cut()
L = sortList(L)
for(var/I in L)
	_symptoms += L[I]
```

or, in a more optimzed way, replace the symptoms += S lines in AddSymptom() with a sorted insert. e.g.:

```
var/inserted = FALSE
var/list/_symptoms = symptoms
for(var/I in 1 to _symptoms.len)
	var/datum/symptom/OS = _symptoms[I]
	if(sortTextEx(OS.id, S.id) == 1)
		_symptoms.Insert(I, S)
		inserted = TRUE
		break
if(!inserted)
	symptoms += S
```